mithril-common/src/cardano_transaction_parser.rs

```diff
@@ -105,16 +105,18 @@ impl Block {
 /// Cardano transaction parser
 pub struct CardanoTransactionParser {
     logger: Logger,
+    /// When set to true, no error is returned on unparsable blocks.
+    /// It can occurs because of the crate 'pallas-hardano'
+    /// that doesn't support all encoding versions (test networks doesn't work)
```

Comment:
```suggestion
    /// When set to true, no error is returned in case of unparsable block, and an error log is written instead.
    /// This can occur when the crate 'pallas-hardano' doesn't support some non final encoding for a Cardano era.
    /// This situation should only happen on the test networks and not on the mainnet.
```